pom.xml

```diff
@@ -109,17 +109,17 @@
 
     <maven.plugin.antrun.version>3.0.0</maven.plugin.antrun.version>
     <maven.plugin.clean.version>3.2.0</maven.plugin.clean.version>
-    <maven.plugin.compiler.version>3.11.0</maven.plugin.compiler.version>
+    <maven.plugin.compiler.version>3.12.1</maven.plugin.compiler.version>
     <maven.plugin.dependency.version>3.1.1</maven.plugin.dependency.version>
     <maven.plugin.flatten.version>1.3.0</maven.plugin.flatten.version>
-    <maven.plugin.jacoco.version>0.8.7</maven.plugin.jacoco.version>
+    <maven.plugin.jacoco.version>0.8.11</maven.plugin.jacoco.version>
     <maven.plugin.jar.version>3.0.2</maven.plugin.jar.version>
     <maven.plugin.os.version>1.7.0</maven.plugin.os.version>
     <maven.plugin.protobuf.version>0.6.1</maven.plugin.protobuf.version>
     <maven.plugin.rat.version>0.13</maven.plugin.rat.version>
-    <maven.plugin.scala.version>4.7.2</maven.plugin.scala.version>
+    <maven.plugin.scala.version>4.8.1</maven.plugin.scala.version>
```

Comment:
I roughly remember there are some issues in 4.8.1, if possible, please choose a version that verified by a released Spark version